engine/src/player-legacy.cpp

```diff
@@ -218,6 +218,8 @@ MCPlayer::MCPlayer()
 	userCallbackStr = NULL;
 	formattedwidth = formattedheight = 0;
 	loudness = 100;
+	dontuseqt = MCdontuseQT;
```

Comment:
This isn't quite what I suggested.

dontuseqt is initialized to false - if it is false at the point a movie is prepared *then* the value of MCdontuseqt is used